### DISCUSSION TITLE
error while loading shared libraries: libiomp5.so: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by goodman249** [firemodels/fds#12648] [General]
When using fds6.7.4 version to simulate computing jobs on the supercomputing center, an error is reported as follows. 
 
error while loading shared libraries: libiomp5.so: cannot open shared object file: No such file or directory

May I ask how to solve it

**Comment by mcgratta** [comment]
```
There is a dependency on the libiomp5.so module. Check to see if your machine already has this module installed by running $find / -name libiomp5.so and if the module is not found, then install it and put it in the modules folder that your program is looking in.
```
I found this by doing a quick Internet search. You should probably talk to the person who administers the computer. Also, it is best to use the latest version of FDS if you want help from the community.
